### PR TITLE
enhancement: fleet and order changes

### PR DESCRIPTION
**Pull request by Daljit-fleetHawks**
## Change Description

Some changes in fleet module components like assets/logs and also done changes on fields mandatory fields in order

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist: (Check only applicable)

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/app/pages/fleet/assets/asset-detail/asset-detail.component.ts [hunk 1]
```diff
@@ -155,6 +155,7 @@ export class AssetDetailComponent implements OnInit {
           if (!this.assetData.hasOwnProperty('devices')) {
             this.assetData['devices'] = [];
           }
+          console.log('this.assetData', this.assetData);
```
Comment (kunalfleethawks) [inline src/app/pages/fleet/assets/asset-detail/asset-detail.component.ts]: Remove console.log

File: src/app/pages/fleet/assets/asset-detail/asset-detail.component.ts [hunk 2]
```diff
@@ -216,6 +217,7 @@ export class AssetDetailComponent implements OnInit {
               name: x,
             }));
           }
+          console.log('photos', this.assetsImages)
```
Comment (kunalfleethawks) [inline src/app/pages/fleet/assets/asset-detail/asset-detail.component.ts]: Remove this

File: src/app/pages/fleet/maintenance/service-log/add-service/add-service.component.ts [hunk 3]
```diff
@@ -128,7 +128,7 @@ export class AddServiceComponent implements OnInit {
    * Add new asset
    */
   addService() {
-    
+    console.log('serviceData', this.uploadedPhotos, this.uploadedDocs);
```
Comment (kunalfleethawks) [inline src/app/pages/fleet/maintenance/service-log/add-service/add-service.component.ts]: Remove this

File: src/app/pages/fleet/maintenance/service-log/add-service/add-service.component.ts [hunk 4]
```diff
@@ -361,7 +360,9 @@ export class AddServiceComponent implements OnInit {
   addParts() {
     
     this.inventory.forEach(element => {
-      if (element.name === this.selectedParts[this.selectedParts.length - 1].name) {
```
Comment (kunalfleethawks) [inline src/app/pages/fleet/maintenance/service-log/add-service/add-service.component.ts:364]: Remove this

**Review by kunalfleethawks — APPROVED** [review]
Please remove console logs statement

**Review by amritfleethawks — APPROVED** [review]

**Review by Daljit-fleetHawks — COMMENTED** [review]
changes done

**Pull request merged**